omod/src/main/webapp/patientcolors.jsp

```diff
@@ -1,21 +1,42 @@
-<%@ include file="/WEB-INF/template/include.jsp"%>
+<DOCTYPE html>
+<html>
+<head>
+ <title>patient colors information</title>
+ 
+</head>
 
-<%@ include file="/WEB-INF/template/header.jsp"%>
+<body>
+<form action="/openmrs-standalone/module/patientcolors/patientcolors.form" method="post">
+<fieldset>
+<legend><b>Enter patient details</b></legend>
+Select patientName:
+<select name = "patient_id">
+<c:forEach var="patient" items"${patients}">
+<option value="${patient.patient_id}">${patient.personName}</option>
```

Comment:
Shouldn't `${patient.patient_id}` be `${patient.patientId}` instead?

Reply:
Also change `items"${patients}` to `items="${patients}`

Reply:
thanks bro 
i made the change but still got the same results